spec/factories.rb

```diff
@@ -40,6 +40,6 @@
     body { Faker::Lorem.paragraph }
     course_id { comment_thread.course_id }
     commentable_id { comment_thread.commentable_id }
-    endorsed false
+    endorsed { false }
   end
 end
```

Comment:
Updates tot accommodate new versions of Faker, and migration from FactoryGirl -> FactoryBot